src/components/NoteDetails.js

```diff
@@ -0,0 +1,193 @@
+const Notes = {
+  '16': {
```

Comment:
Why use numbers as keys?

Reply:
Also, there seems to be a lot of duplication on `noteName` and `string`, can you reuse the same property? If not, check that all values are as intended, there seem to be a few where noteName does not match string

Reply:
Let's talk about this tomorrow. I have answers but this also spurred ideas.